.changes/next-release/bugfix-s3-90647.json

```diff
@@ -0,0 +1,5 @@
+{
+  "type": "bugfix",
+  "category": "s3",
```

Comment:
Use \`\`s3\`\` for `category` here so it displays as inline code in the rendered RST. Also make this change in the PR for the `v2` branch.